owl/EASE-ACT.owl

```diff
@@ -1,18 +1,18 @@
 <?xml version="1.0"?>
 <rdf:RDF xmlns="http://www.ease-crc.org/ont/EASE-ACT.owl#"
      xml:base="http://www.ease-crc.org/ont/EASE-ACT.owl"
-     xmlns:ease="http://www.ease-crc.org/ont/EASE.owl#"
-     xmlns:ease-proc="http://www.ease-crc.org/ont/EASE-PROC.owl#"
-     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:owl="http://www.w3.org/2002/07/owl#"
+     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xml="http://www.w3.org/XML/1998/namespace"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
-     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#">
+     xmlns:ease="http://www.ease-crc.org/ont/EASE.owl#"
+     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+     xmlns:ease-proc="http://www.ease-crc.org/ont/EASE-PROC.owl#">
     <owl:Ontology rdf:about="http://www.ease-crc.org/ont/EASE-ACT.owl">
-        <owl:imports rdf:resource="package://ease_ontology/owl/EASE-PROC.owl"/>
-        <owl:imports rdf:resource="package://ease_ontology/owl/EASE-STATE.owl"/>
+        <owl:imports rdf:resource="http://www.ease-crc.org/ont/EASE-PROC.owl"/>
```

Comment:
This is problematic, I am afraid.

First of all, the files are still not there, http://www.ease-crc.org/ont/EASE-PROC.owl gives 404. Second, ease-crc returns a HTML page. I experienced problems clients not detecting this and trying to parse the HTML as OWL.
I know, this should be fixed.

Anyway, I was thinking that this can be ignored, as only other EASE ontologies are imported that way. Hence they do not need to appear in the uglified version. Then you could leave this hack for now.

Reply:
The `package:` variant also fails to resolve and is confusing if one wants to use the ontology without a ROS stack.
With this URL, at least manual debugging will help figuring out that the files give 404s.

Nevertheless, I can also change the URLs to match a resource where the files are available, e.g. GitHub. I could also try and change this PR in a way that it automatically builds the gh-pages branch with all resulting ontologies and serves them as https://ease-crc.github.io/ease_ontology/owl/EASE.owl etc.

Reply:
Yes it is confusing. But we cannot break KnowRob. The package URLs must remain until we have a proper solution.